addons/website/static/src/js/backend/button.js

```diff
@@ -40,5 +41,63 @@ var WidgetWebsiteButton = AbstractField.extend({
     },
 });
 
+var WidgetWebsiteButtonIcon = AbstractField.extend({
+    template: 'WidgetWebsiteButtonIcon',
+    fieldDependencies: _.extend({}, AbstractFieldBinary.prototype.fieldDependencies, {
```

Comment:
Why are using `AbstractFieldBinary` ? :hushed:

Reply:
copy paste got me

Reply:
You should have seen that before sending to review :wink: